SolarSystem.pde

```diff
@@ -296,12 +297,14 @@ void explode(Planet p, int i) {
   for (int j=0; j < pieces; j++) {
     newMasses[j] *= massFac;
     newRadii[j] *= sqrt(areaFac);
-    if (pieces-1 != j) {
-      newVels[j] = PVector.random2D().mult(newMasses[j] * newVelMags[j]);
-      newVels[pieces-1].sub(newVels[j]);
-    }
+    newVels[j] = PVector.random2D().mult(newMasses[j] * newVelMags[j]);
```

Comment:
Could you normalize this random2D vector? It generates a vector between -1 and 1 on both axes, so, on average, it'll turn into a square.